src/views/repos/show.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import GraphEditor from '../../components/graph_editor/flux/components/graph_editor_base'
+import ampersandMixin from 'ampersand-react-mixin'
+import Icon from'react-fa'
+
+export default React.createClass({
+  mixins: [ampersandMixin],
+
+  displayName: 'RepoDetailPage',
+
+  render () {
+    let repo = <Icon name="spinner"/>
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>Expected an identifier and instead saw '<'.<br>Missing semicolon.<br>Expected an assignment or function call and instead saw an expression.<br>Mixed double and single quotes.<br>Read only.<br>Expected an identifier and instead saw '>'.